### PR TITLE
Updated the starter kit for @fastly/as-compute 0.2.0

### PR DESCRIPTION
**Pull request by torch2424**
This updates the starter kit to work with the next update for `@fastly/as-compute`, which has some breaking changes, that better match the [Fetch API, for example the Request Object should have properties, and not functions](https://developer.mozilla.org/en-US/docs/Web/API/Request). :smile: 

It also updates the `package.json`, with all the most recent versions of AssemblyScript and `@fastly/as-compute`.

Screenshot from 2021-01-07 12-07-14

### CODE REVIEW

**Comment by torch2424** [comment]
I meant to open this as a Draft PR :upside_down_face: I'll ping you once the new version is live :smile:

**Review by phamann — APPROVED** [review]
:+1: LGTM

**Comment by phamann** [comment]
@torch2424 Looks like you also need to push the changes to the package-lock.json lockfile as thats causing CI to fail as we use `npm ci`.

**Comment by torch2424** [comment]
So, `@fastly/as-compute@0.2.0` is live! :smile: https://www.npmjs.com/package/@fastly/as-compute / https://github.com/fastly/ExecuteD/pull/740

The new docs are available here: https://as-compute-reference-docs.edgecompute.app/

Essenitally, I just updated everything for the breaking change of the ES Getter/Setter functions as mentioned in my inital post :smile: 

Here is a changelog of the two packages:

* https://unpkg.com/browse/@fastly/as-compute@0.2.0/CHANGELOG.md
* https://unpkg.com/browse/@fastly/as-fetch@0.2.0/CHANGELOG.md

No rush here, since everything is versioned, I won't let our current users know (in the support slack) until everything is up to date :smile: . I also will start the review for the devhub: https://github.com/fastly/Developer-Portal/pull/489

---

And per your comment @phamann , my apologies! Yep, I intentionally didn't include the lockfile, because 0.2.0 wasn't published yet, so it would break. But now it is, so now it can be installed, and everything works fine! :smile: 

Screenshot from 2021-01-21 12-12-05

I'll wait for one more approval from you if you don't mind, and then feel free to merge this! :smile: :tada:

**Pull request merged**
